spec/support/checkins_spec_helpers.rb

```diff
@@ -1,7 +1,11 @@
 module CheckinsSpecHelpers
   def call_checkin_action(method, number, checkin)
-    get method.to_sym, params: params
-    check_response_hash(number, checkin)
+    get method.to_sym, params: params, format: "json"
+    if number.zero?
```

Comment:
Root element of response body has nothing inside. That's why I had to add this check.